### PR TITLE
balancer: add metrics.

### PR DESCRIPTION
**Pull request by disksing**
For letting us know why a cluster is not well balanced.

### CODE REVIEW

**Comment by siddontang** [comment]
LGTM

File: server/metrics.go [hunk 1]
```diff
@@ -65,6 +65,14 @@ var (
 			Help:      "Status of the scheduler.",
 		}, []string{"kind", "type"})
 
+	balancerCounter = prometheus.NewCounterVec(
```
Comment (huachaohuang) [inline server/metrics.go]: Seems it collect metrics from other schedulers too, maybe rename to `schedulerCounter`?

Comment (disksing) [inline server/metrics.go]: ok.

File: server/metrics.go [hunk 2]
```diff
@@ -65,6 +65,14 @@ var (
 			Help:      "Status of the scheduler.",
 		}, []string{"kind", "type"})
 
+	schedulerCounter = prometheus.NewCounterVec(
+		prometheus.CounterOpts{
+			Namespace: "pd",
+			Subsystem: "scheduler",
+			Name:      "banalcer",
```
Comment (huachaohuang) [inline server/metrics.go]: Need to change name and help too.
Rest LGTM.

**Comment by siddontang** [comment]
PTAL @huachaohuang

**Comment by huachaohuang** [comment]
LGTM

**Pull request merged**